app/services/dttp/retrieve_account.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveAccount
+    include ServicePattern
+
+    class HttpError < StandardError; end
+
+    attr_reader :provider
+
+    ACCOUNT_ATTRIBUTE_FILTERS = %w[
+      name
+      accountid
```

Comment:
I believe `accountid` is returned by default, so you don't need to include it in the select.